CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
```

Comment:
This is a limitation with how we are including the changelog. Hugo automatically adds a title, but we also include one in the file itself. There is likely a way to resolve this, but since it is not a big problem, I'm going to hold off on it for now.